client/reader/embed-helper.jsx

```diff
@@ -87,6 +92,10 @@ function resolveEmbedConfig( embed ) {
 	return embedsConfig.default;
 }
 
+function isPercentage( val ) {
+	return 'string' === typeof val && '%' === val[ val.length - 1 ];
```

Comment:
might want a `val && ` to start this statement so you don't try do index into -1 on an empty string